doc/nrf/templates/sample_README.rst

```diff
@@ -61,34 +85,128 @@ Connect PIN1 to PIN2, then cut the connection again.
 
 User interface*
 ***************
+.. note::
+   Add button and LED assignments here, plus other information related to the user interface.
 
 Button 1:
-   Start or stop the disco show.
+   Does something.
 
-.. tip::
-   Add Button and LED assignments here, plus other information related to the user interface.
+Button 2:
+   Toggles something.
+
+LED 1:
+   On when connected.
+
+LED 2:
+   Indicates something.
+
+
+Configuration*
+**************
+
+.. note::
+   Always include this section if your sample can be configured by the user.
+   Start with the stock phrase that is included with ``|config|``.
+
+|config|
+
+Setup*
+======
+
+.. note::
+   Add general setup information here, for example, that the user must install or enable some library or select a specific backend.
+   Most samples do not need this section.
+
+Configuration options*
+======================
+
+.. note::
+   * You do not need to list all configuration options of the sample, but only the ones that are important for the sample and make sense for the user to know about.
+   * The syntax below allows to link to sample configuration options in the same way as library configuration options (``:option:`SAMPLE_CONFIG```, which results in :option:`SAMPLE_CONFIG`).
+   * For each configuration option, list the symbol name and the string describing it.
+
+Check and configure the following configuration options for the sample:
+
+.. option:: SAMPLE_CONFIG - Sample configuration
+
+   The sample configuration defines ...
+
+.. option:: ANOTHER_CONFIG - Another configuration
+
+   This configuration option specifies ...
+
+
+Additional configuration*
+=========================
+
+.. note::
+   * Add this section to describe and link to any library configuration options that might be important to run this sample.
+     You can link to options with ``:option:`CONFIG_XXX```.
+   * You do not need to list all possible configuration options, but only the ones that are relevant.
+
+Check and configure the following library options that are used by the sample:
+
+* :option:`CONFIG_BT_DEVICE_NAME` - Defines the Bluetooth device name.
+* :option:`CONFIG_BT_GATT_LBS` - Enables the :ref:`lbs_readme`.
+
+Configuration files*
+====================
+
+.. note::
+   Add this section if the sample provides predefined configuration files.
+
+The sample provides predefined configuration files for typical use cases.
+You can find the configuration files in the :file:`XXX` directory.
+
+The following files are available:
+
+* :file:`filename.conf` - Specific scenario.
+  This configuration file ...
 
 Building and running
```

Comment:
Thread and Zigbee samples also include a line about enabling the stack before testing (+ link to stack configuration page).

Reply:
But the samples are configured to include the stack by default, or aren't they? If they are NOT configured correctly by default, this information should go into the Setup section.
If they are configured correctly by default, I think that line should be moved to after the build_and_run.txt and be reworded to something like "Note that the sample enables the XXX stack. See \<link to UG>." I added that line.